CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.11.7
```

Comment:
Since this is not a functional change, we shouldn't bump the package version

Reply:
Ok, that makes sense. And should I add a note about adding the rule to the changelog (to the current version) or we don't need the note there?

Reply:
No need for a note in the changelog. The changelog is intended for consumers of the library.